go/server/handlers.go

```diff
@@ -20,21 +20,48 @@ package server
 
 import (
 	"github.com/gin-gonic/gin"
+	"github.com/vitessio/arewefastyet/go/tools/macrobench"
 	"github.com/vitessio/arewefastyet/go/tools/microbench"
 	"log"
 	"net/http"
 	"sort"
 )
 
+func handleRenderErrors(c *gin.Context, err error) {
+	if err == nil {
+		return
+	}
+	log.Println(err.Error())
+	c.HTML(http.StatusOK, "error.tmpl", gin.H{
+		"title":    "Vitess benchmark - Error",
+		"url":      c.FullPath(),
+		"errorStr": err.Error(),
```

Comment:
we should not give the internal error out. It should be some masked error message.

Reply:
Thank you, that's a good suggestion. I went ahead and changed the way to render errors in the corresponding template. I have removed the `errorStr` field and used a static text instead. The error itself is still logged serverside. Fix was pushed in 3386b9122f9d8dd584c8af41f12e22204a09da22.


![image](https://user-images.githubusercontent.com/35779988/113771301-7c4cf580-9723-11eb-99ad-0f3dd097bec2.png)